src/shared/components/post/post-listing.tsx

```diff
@@ -103,7 +103,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
   private readonly isoData: IsoDataOptionalSite = setIsoData(this.context);
   state: PostListingState = {
     showEdit: false,
-    imageExpanded: !!this.isoData.site_res?.site_view.site.content_warning,
```

Comment:
Images are supposed to auto expand when content warning is present.

Reply:
Not the opposite? Collapsed while the modal is showing. Expanded when auto expand is enabled and the image is not nsfw or the user settings disable the blur for nsfw.

Reply:
I'm using lemmynsfw's behavior as a reference, but TBH this does seem like something that should be a separate site setting.

Reply:
related to this: https://github.com/LemmyNet/lemmy/issues/4643

I'd prefer we just let the new `local_site.default_post_listing_mode` handle this, and add a card view.